tools/common/StaticRegistrar.cs

```diff
@@ -2391,26 +2429,38 @@ static bool IsTypeCore (ObjCType type, string nsToMatch)
 		static bool IsMetalType (ObjCType type)
 		{
 			var ns = type.Type.Namespace;
-			if (!IsDualBuild)
+#if MONOMAC
+			if (!IsUnified)
 				ns = ns.Substring (CompatNamespace.Length + 1);
+#endif
 
 			switch (ns) {
-			case "Metal":
-			case "MetalKit":
+			//case "Metal":
+			//case "MetalKit":
```

Comment:
is that because they are now in macOS ?
if so this looks like something we should fix right away, c.c. @chamons